contributors.html

```diff
@@ -723,6 +690,36 @@ <h3 class="g-font-size-18--xs g-color--white">mozilla kerala</h3>
 document.getElementsByTagName('body')[0].appendChild(idl);
 })();
 </script>
+<script>
+    window.fbAsyncInit = function() {
```

Comment:
Should we really track the page views in fb? @riverspirit @shinenelson any thoughts?

Reply:
will remove the tracking code

Reply:
removed the tracking code.